models/Pet.js

```diff
@@ -9,18 +9,15 @@ Pet.init(
             type: DataTypes.STRING,
             allowNull: false,
         },
-        // TODO: change datatypes?
         age: {
-            type: DataTypes.DECIMAL,
+            type: DataTypes.STRING,
```

Comment:
Please make 'INTEGER'

Reply:
I kept STRING BECAUSE lots of pet are only few months old and lots of pets only live couple years, so STRiNG would allow to write more details.